firestore.rules

```diff
@@ -1,7 +1,11 @@
+rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /{document=**} {
-      allow read, write: if false;
+    match /apiKeys/{document=**} {
+      allow read, write: if request.auth.token.email.split("@")[1] == "actnowcoalition.org";
+    }
+    match /share-links/{document=**} {
+    	allow read, write: if false;
```

Comment:
I'm assuming this is the default and so you don't need to explicitly specify, but 🤷